### PR TITLE
Midsummer site refresh

### PR DESCRIPTION
**Pull request by mikol**

### CODE REVIEW

File: site/index.md [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+{: .opa-homepage--section--feature--text}
+
+</div>
+
+<div class="opa-homepage--section--feature" markdown="1">
+
+### Embeddable
+{: .opa-homepage--section--feature--title}
+
+Anyone can use OPA’s interactive shell to quickly experiment with queries and data sets.
```
Comment (tsandall) [inline site/index.md]: Looks like this got copied from above.

File: site/get-opa/index.md [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+
+# Get Open Policy Agent
+{: .opa-header--minor-title}
+
+OPA is available as binary releases for 64-bit Linux and Mac OS X or as Golang source code.
+{: .opa-header--text}
+
+  * [64-bit Linux](https://github.com/open-policy-agent/opa/releases/download/v0.1.0-rc3/opa_linux_amd64){: .opa-header--download-list--link}
+  * [64-bit Mac OS X](https://github.com/open-policy-agent/opa/releases/download/v0.1.0-rc3/opa_darwin_amd64){: .opa-header--download-list--link}
+  * [Go Source](https://github.com/open-policy-agent/opa/releases/download/v0.1.0-rc3/opa0.1rc3.src.tar.gz){: .opa-header--download-list--link}
```
Comment (tsandall) [inline site/get-opa/index.md]: Link: https://github.com/open-policy-agent/opa/archive/v0.1.0-rc3.tar.gz

File: site/examples/working-with-the-opa-repl/index.md [hunk 3]
```diff
@@ -0,0 +1,270 @@
...
+
+# Working with the OPA REPL
+
+REPLs are great for learning new languages and running quick experiments. You can take advantage of OPA's REPL to quickly run ad-hoc queries or prototype policies.
+
+{% endcontentfor %}
+
+{% contentfor body %}
+
+## OPA’s REPL
```
Comment (tsandall) [inline site/examples/working-with-the-opa-repl/index.md]: Do we want to title this Goals to make it consistent with the Docker Authorization example?

File: site/documentation/how-does-opa-work/index.md [hunk 4]
```diff
@@ -0,0 +1,512 @@
...
+{% img '{{assets["data-model-dependencies.svg"].logical_path}}' width:'720' %}
+
+### Base Documents
+
+So-called base documents contain static, structured data stored in memory and optionally saved to disk for resiliency. Your service will publish and update base documents in order to describe its current state, and your users can do the same to include relevant data about the state of their own deployment context.
+
+Base documents are published and updated using OPA’s Data API. For example, the following request publishes a list of servers to OPA:
+
+```http
+PATCH /v1/data/servers HTTP/1.1
```
Comment (tsandall) [inline site/documentation/how-does-opa-work/index.md]: Other examples are using https://example.com/ in the URL. Either way is fine by me.

File: site/documentation/how-does-opa-work/index.md [hunk 5]
```diff
@@ -0,0 +1,512 @@
...
+
+```http
+GET https://example.com/v1/data/opa/examples/violations HTTP/1.1
+```
+
+### Putting It All Together
+
+Let’s take a look at some documents representing the state of a hypothetical service and a policy module that uses this data. The following documents describe a set of servers, the protocols they use, the ports they open, and the networks those ports are connected to.
+
+```json
```
Comment (tsandall) [inline site/documentation/how-does-opa-work/index.md]: Just for future reference, I think a small illustration of servers/nics/networks could be useful here.

File: site/documentation/how-do-i-write-policies/index.md [hunk 6]
```diff
@@ -21,34 +28,34 @@ Use Rego for defining policy that is easy to read and write.
...
+Rego is declarative so policy authors can focus on what queries should return rather than how queries should be executed. These queries are simpler and more concise than the equivalent in an imperative language. Like other applications which support declarative query languages, OPA is able to optimize queries to improve performance (for example, indexing, concurrent evaluation, reordering, and so on).
 
 ## The Basics
 
 This section introduces the main aspects of Rego.
 
 The simplest rule is a single expression and is defined in terms of a [Scalar Value](#scalar-values):
 
-```rego
+```ruby
```
Comment (tsandall) [inline site/documentation/how-do-i-write-policies/index.md]: Nice.

File: site/community/index.md [hunk 7]
```diff
@@ -0,0 +1,28 @@
...
+{: .opa-header--subtitle}
+
+Open Policy Agent (OPA) is an open source policy engine. Contributions are always welcome. <span class="opa-keep-it-together">For information about contributing, see [CONTRIBUTING.md](https://github.com/open-policy-agent/opa/blob/master/CONTRIBUTING.md){: .opa-header--link}.</span>
+{: .opa-header--text}
+
+{% endcontentfor %}
+
+{% contentfor body %}
+
+  * [{% img '{{assets["community-slack.svg"].logical_path}}' alt:'Slack: community chat' %}](https://openpolicyagent.slack.com/)
```
Comment (tsandall) [inline site/community/index.md]: We need to deploy a bot to send invites to people who want to use the OPA slack organization.

I think we should wait for the bot before including the link here and in the footer.

**Comment by tsandall** [comment]
LG(reat)TM! A couple minor comments about points I noticed while browsing through the site on my machine.

File: site/index.md (hunk 1 above)
Comment (mikol) [inline site/index.md]: Done.

File: site/get-opa/index.md (hunk 2 above)
Comment (mikol) [inline site/get-opa/index.md]: Thanks! I should have mentioned that we needed this link. What would it take to have downloads for Linux and Mac named “opa” without needing the rename step in the curl instructions below? Should we just publish a binary to a directory like  `releases/download/v0.1.0-rc3/linux_amd64/opa`?

File: site/examples/working-with-the-opa-repl/index.md (hunk 3 above)
Comment (mikol) [inline site/examples/working-with-the-opa-repl/index.md]: Yes. That is an excellent idea.

File: site/documentation/how-does-opa-work/index.md (hunk 4 above)
Comment (mikol) [inline site/documentation/how-does-opa-work/index.md]: Good catch. Done.

File: site/documentation/how-does-opa-work/index.md (hunk 5 above)
Comment (mikol) [inline site/documentation/how-does-opa-work/index.md]: Okay.

File: site/community/index.md (hunk 7 above)
Comment (mikol) [inline site/community/index.md]: I can add a “Coming Soon” stamp to this item. Sound good?

File: site/get-opa/index.md (hunk 2 above)
Comment (tsandall) [inline site/get-opa/index.md]: I don't think you can do that with releases. I looked at a few other projects that distribute binaries through GitHub releases and this seems to be the model.

File: site/community/index.md (hunk 7 above)
Comment (tsandall) [inline site/community/index.md]: Perfect!

**Comment by tsandall** [comment]
As soon as the fixes are pushed, I'll merge both PRs.

**Comment by mikol** [comment]
Thanks for the comments, @tsandall! I think we are ready to go.

**Pull request merged**